packages/babel-plugin-ember-test-metadata/__tests__/index-test.js

```diff
@@ -2,11 +2,18 @@ const pluginTester = require('babel-plugin-tester').default;
 const addMetadata = require('../index');
 const path = require('path');
 
+const projectInfo = {
```

Comment:
I'd probably call this `pluginConfiguration`, as that's a little more accurate to what it is.

```suggestion
const projectConfiguration = {
```